requirements/edx/base.txt

```diff
@@ -91,6 +91,7 @@ django-ratelimit-backend==0.6
 unicodecsv==0.9.4
 django-require==1.0.6
 pyuca==1.1
+wrapt==1.10.5
```

Comment:
There are not any performance implications of `functools.wraps`(not `setuptools.wraps`, my bad :worried: )
As mentioned above the `wrapt` library wraps decorated functions very well.